lib/htty/cli/display.rb

```diff
@@ -153,11 +153,12 @@ def show_response(response)
     end
     print ' '
     cookies_asterisk = response.cookies.empty? ? '' : strong('*')
-    body_length = response.body.to_s.length
-    body_size   = body_length.zero? ? 'empty' : "#{body_length}-character"
+    body_length      = response.body.to_s.length
+    body_size        = body_length.zero? ? 'empty' : "#{body_length}-character"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.